.gitignore

```diff
@@ -101,3 +101,5 @@ version.ml
 version.texi
 .cargo
 !src/vendor/**
+**/target/rls/debug/
```

Comment:
I just tried and RLS won't even start on VSCode 😂 Manually running `rls` still places the `target/rls` folder next to Cargo.toml. 

```sh
$ cd src/bootstrap
$ rls sfjsdjfl
^C
$ ls target
rls/
```

If RLS respect workspace settings, this line can indeed be removed as it is covered by the next line too.